cogs/info.py

```diff
@@ -16,6 +16,44 @@ def __init__(self, bot):
         self.process = psutil.Process(os.getpid())
 
     ## Proof of concept for use with @story{9}
+    @commands.command()
+    async def trivia (self, ctx, arg1 = "", arg2 = ""):
+
+        # *** BEGIN ***
+        if (arg1.lower() == "begin" or arg1.lower() == "start"):
+            if (self.bot.trivia.inProgress):
+                await ctx.send("Trivia is already in progress")
+            else:
+                await ctx.send("Starting trivia...")
+                if (self.bot.trivia.start() == 0):
+                    self.bot.trivia.inProgress = True
+
+        # *** END ***
+        elif (arg1.lower() == "end" or arg1.lower() == "stop"):
+            if (self.bot.trivia.inProgress):
+                await ctx.send("Stopping trivia...")
+                if (self.bot.trivia.end() == 0):
+                    self.bot.trivia.inProgress = False
+            else:
+                await ctx.send("Trivia is not in progress")
+
+        # *** STATUS ***
```

Comment:
This commenting style looks a little strange to me, but it definitely does stand out from other comments, which is a good thing.